modules/apps/object/object-api/src/main/java/com/liferay/object/exception/ObjectEntryValuesException.java

```diff
@@ -120,10 +120,13 @@ public static class ExceedsMaxFileSize extends ObjectEntryValuesException {
 
 		public ExceedsMaxFileSize(long maxFileSize, String objectFieldName) {
 			super(
+				Arrays.asList(maxFileSize, objectFieldName),
 				String.format(
 					"File exceeds the maximum permitted size of %s MB for " +
 						"object field \"%s\"",
-					maxFileSize, objectFieldName));
+					maxFileSize, objectFieldName),
+				"the-entry-value-exceeds-the-maximum-length-of-x-characters-" +
```

Comment:
Hi @pedro-oliveira446 , should the key be about the file size in MB?

Reply:
Hey @linolaoj,
you're right, I just realized this message didn't exist yet, so I created a new one.